documentation/bolt_examples.md

```diff
@@ -123,3 +123,228 @@ To convert the `restart_service.ps1` script to a task, giving you the ability to
     ![](bolt_PS_3.png)
 
     **Note:** `--nodes windows` refers to the name of the group of target nodes that you specified in your inventory file. For more information, see [Specify target nodes](bolt_options.md#).
+
+
+## Deploy a package with Bolt and Chocolatey
+
+You can use Bolt with Chocolatey to deploy a package on a Windows node by writing a Puppet plan that installs Chocolatey and uses the Chocolatey package provider to install a package. This is all done using content from the Puppet Forge.
+
+**Before you begin** 
+
+- Install Bolt and Puppet Development Kit (PDK).
+- Ensure you have Powershell and Windows Remote Management (WinRM) access.
+
+In this example, you:
+
+- Build a project-specific configuration using a Bolt project directory and PDK.
+- Download module content from the Puppet Forge.
+- Write a Bolt plan to apply Puppet code and orchestrate the deployment of a package resource using the Chocolatey provider.
+
+### 1. Build a project-specific configuration using a Bolt project directory Bolt runs in the context of a project directory. This directory contains all of the configuration, code, and data loaded by Bolt.
+    
+1. Create a module using the PDK command `pdk new module puppet_choco_tap` 
+
+2. To make the module a Bolt project directory, add a `bolt.yaml` file. 
+
+3. Add the following configuration to the `bolt.yaml` file. 
+
+```
+Modulepath: "./modules/:~/.puppetlabs/bolt-code/modules:~/.puppetlabs/bolt-code/site-modules"
```

Comment:
This seems wrong. Why is `Modulepath` capitialized? I'm thinking maybe this appears to work becuase `Modulepath` gets ignored. It looks like everything should be contained within the `puppet_choco_tap` module (bolt project dir) in which case all you would need is an empty `bolt.yaml`. So setting the modulepath would be unnecessary and i *think* would not find the project specific plan.

Reply:
@donoghuc Do you think this whole step should be removed?

Reply:
Yeah, it is not needed. I just ran through to verify. As it is written Modulepath is ignored.

Reply:
that's just where the conversion from google doc as picked up the capitalisation it should be lowercase, it is needed for the example workflow.